Tools/Scripts/webkitpy/performance_tests/perftestsrunner.py

```diff
@@ -122,8 +118,10 @@ def _expand_path(option, opt_str, value, parser):
                 help="Don't launch a browser with results after the tests are done"),
             optparse.make_option("--test-results-server",
                 help="Upload the generated JSON file to the specified server when --output-json-path is present."),
-            optparse.make_option("--dump-render-tree", "-1", action="store_false", default=True, dest="webkit_test_runner",
-                help="Use DumpRenderTree rather than WebKitTestRunner."),
+            optparse.make_option("--dump-render-tree", "-1", action='append_const', dest='driver_names', const="DumpRenderTree", default=[],
```

Comment:
Nit: `default=[]` is redundant here (it's the default default with `append_const`).

Reply:
It's not redundant in `optparse` with however our tests work, sadly.